docs/ossm/create-mesh/README.md

```diff
@@ -0,0 +1,121 @@
+# Scoping the service mesh with DiscoverSelectors
+This page describes how the service mesh control plane monitors cluster resources and how to manage its scope. 
+
+A service mesh will include a workload that:
+1. Has been discovered by the service mesh control plane
+1. Has been [injected with a Envoy proxy sidecar](../injection/README.md)
+
+
+By default, the service mesh control plane will watch all namespaces within the cluster, meaning that:
+- Each proxy instance will receive configuration for all namespaces. This includes information about workloads that are not enrolled in the mesh.
+- Any workload with the appropriate sidecar or namespace injection label will be injected with a proxy sidecar.
+
+This may not be desirable in a shared cluster, and you may want to limit the scope of the service mesh to only a portion of your cluster. This is particularly important if you plan to have [multiple service meshes within the same cluster](./multi-control-planes/README.md).
```

Comment:
Why is this documentation separate from the multi-control-plane documentation? Is it to highlight two different use cases for discovery selectors? Is this documentation _only_ using discovery selectors for performance reasons?

Reply:
They are different use cases. I see this section as being about isolating an individual mesh on a cluster - roughly analogous to what SMMR did, but with only one mesh in the cluster. This is probably the 80-90% of users who aren't interested in multiple control planes but don't want the mesh to impact other namespaces. 

After seeing the two mesh example, I think it'd be confusing for your average user getting started with just one mesh to unpack. You could combine them into one set of examples, but I think the multi-control plane case deserves a bit more context and consideration if it's even a good idea (compared to single control plane multi-tenancy), which I'm planning to write.